cmd/notation/plugin.go

```diff
@@ -29,10 +30,17 @@ func pluginListCommand() *cobra.Command {
 }
 
 func listPlugins(command *cobra.Command) error {
-	mgr := manager.New()
-	plugins, err := mgr.List(command.Context())
+	mgr := plugin.NewCLIManager(dir.PluginFS())
+	pluginNames, err := mgr.List(command.Context())
 	if err != nil {
 		return err
 	}
-	return ioutil.PrintPlugins(os.Stdout, plugins)
+	var plugins []plugin.Plugin
+	var errors []error
+	for _, n := range pluginNames {
+		pl, err := mgr.Get(command.Context(), n)
+		errors = append(errors, err)
+		plugins = append(plugins, pl)
+	}
+	return ioutil.PrintPlugins(command.Context(), os.Stdout, plugins, errors)
```

Comment:
@patrickzheng200 We need to revisit the logic of this part since it does not print the plugins proper if there is any error.